src/manager.js

```diff
@@ -0,0 +1,100 @@
+import Cohort from './cohort.js'
+import { localeIncludes } from 'locale-includes'
+export default class Manager {
+  constructor() {
+    this.cohorts = []
+    this.cohortId = 0
+    this.studentId = 0
+  }
+
+  addCohort(name) {
+    if (name.length === 0) return 'Cohort needs to have name'
+    if (this.cohorts.some((cohort) => cohort.name === name))
```

Comment:
That is a smart way to figure out if there are overlapping cohort names! I'm definitely going to implement that to my code

Reply:
I like this as well, so simple and in one line.